client/app/scripts/utils/router-utils.js

```diff
@@ -8,13 +8,18 @@ import { route } from '../actions/app-actions';
 //
 const SLASH = '/';
 const SLASH_REPLACEMENT = '<SLASH>';
+const PERCENT = '%';
+const PERCENT_REPLACEMENT = '<PERCENT>';
 
 function encodeURL(url) {
-  return url.replace(new RegExp(SLASH, 'g'), SLASH_REPLACEMENT);
+  return url
+    .replace(new RegExp(PERCENT, 'g'), PERCENT_REPLACEMENT)
+    .replace(new RegExp(SLASH, 'g'), SLASH_REPLACEMENT);
 }
```

Comment:
If only there were some sort of standard encoding you could pull off-the-shelf...